.github/CONTRIBUTING.md

```diff
@@ -24,7 +24,7 @@ or if you have ssh set up you can use that URL instead:
 
 ## Commit Style Guide
 
- Mifos Self-Service Android application have a set of [Commit Style Guidelines](https://github.com/openMF/self-service-app/wiki/Commit-Style-Guide). we strictly following this guideline to track every change, any bug fixes, any enhancement, and any new feature addition. we are suggesting you, please follow these guidelines to help us managing every commit.
+ Mifos Self-Service Android application has a set of [Commit Style Guidelines](https://github.com/openMF/self-service-app/wiki/Commit-Style-Guide). we strictly following this guideline to track every change, any bug fixes, any enhancement, and any new feature addition. we are suggesting you, please follow these guidelines to help us manage every commit.
```

Comment:
Correct `we strictly following` , `bug fixes` and other grammatical errors in the same paragraph as well. One more thing, when you handle some docs or any knid of issue to do finish it till end, e.g., in this grammatical error fixing issue, you should go through the whole file and eradicate all such errors.

Reply:
I have not found any other error apart of this